### DISCUSSION TITLE
Streets of rage 4

### DISCUSSION THREAD

**Discussion by masterpiece67** [skyline-emu/skyline#1316] [Game Requests]
![Screenshot_2022-05-15-20-20-14-759_skyline emu](https://user-images.githubusercontent.com/92961132/168472750-23cd4faf-ab32-4ab9-b16b-68e6daa32143.jpg)

I tried to test the game streets of rage 4, and finally it can booting. so can you guys improve this game. I hope the skyline in the future can be even better

**Comment by kirky2k2** [comment]
I'm not downloading another version of skyline till they get this to at least boot into the game

**Reply by Roydpeter** [reply]
Oh no, what will they do without one of your downloads! It's a tragedy I tell you

**Reply by yhak22** [reply]
well now it boot ingame and kinda playable but there is graphical issues , maybe they will fix it soon ;)

**Comment by yhak22** [comment]
this game will come to android on july dude
you can now pre register on googleplay.

**Reply by kirky2k2** [reply]
Actually it's set for release on 24th May on android

**Reply by yhak22** [reply]
oh thanks i mistake the date with another game

**Reply by ghost** [reply]
dudes pirating games tho so pointless of you to assume he will buy it.

**Comment by ghost** [comment]
Lmao

Sent from Yahoo Mail on Android 
 
  On Mon, May 30, 2022 at 11:37 PM, ***@***.***> wrote:   


Oh no, what will they do without one of your downloads! It's a tragedy I tell you
